modules/sno-adding-worker-nodes-to-sno-clusters.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+:_module-type: PROCEDURE
+[id="sno-adding-worker-nodes-to-sno-clusters_{context}"]
+= Adding worker nodes using the Assisted Installer on console.redhat.com
+
+You can add worker nodes to {sno} clusters that were created on link:https://console.redhat.com[console.redhat.com] using the Assisted Installer.
+
+[IMPORTANT]
+====
+Adding worker nodes to {sno} clusters is only supported for clusters running {product-title} version 4.11 and up.
+====
+
+.Prerequisites
+
+* Access to a single-node cluster installed using link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[Assisted Installer].
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Ensure that the worker node is accessible on the same network as the {sno} cluster. A DNS record must exist for the Kubernetes API record names.
+
+.Procedure
+
+. Log in to link:https://console.redhat.com/openshift/assisted-installer/clusters[console.redhat.com] and click on the single-node cluster that you want to add a worker node to.
```

Comment:
should be https://console.redhat.com/openshift ? User cannot add hosts from the AI page

Reply:
I agree, good catch.

Reply:
Updated.